packages/next/src/shared/lib/normalized-asset-prefix.test.ts

```diff
@@ -0,0 +1,33 @@
+import { normalizedAssetPrefix } from './normalized-asset-prefix'
+
+describe('normalizedAssetPrefix', () => {
+  it('should return an empty string when assetPrefix is nullish', () => {
+    expect(normalizedAssetPrefix(undefined)).toBe('')
+  })
+
+  it('should return an empty string when assetPrefix is an empty string', () => {
+    expect(normalizedAssetPrefix('')).toBe('')
+  })
+
+  it('should return an empty string when assetPrefix is a single slash', () => {
+    expect(normalizedAssetPrefix('/')).toBe('')
+  })
+
+  it('should remove leading slash(es) when assetPrefix has more than one', () => {
```

Comment:
```suggestion

  // we expect an empty string because it could be an unnecessary trailing slash
  it('should remove leading slash(es) when assetPrefix has more than one', () => {
```